dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/JobTask.kt

```diff
@@ -14,19 +14,31 @@
  */
 package com.spectralogic.dsbrowser.gui.services.jobService
 
+import com.spectralogic.ds3client.Ds3Client
+import com.spectralogic.ds3client.commands.spectrads3.CancelJobSpectraS3Request
+import com.spectralogic.dsbrowser.api.services.logging.LogType
+import com.spectralogic.dsbrowser.api.services.logging.LoggingService
+import com.spectralogic.dsbrowser.gui.DeepStorageBrowserPresenter
+import com.spectralogic.dsbrowser.gui.services.jobService.factories.removeJob
+import com.spectralogic.dsbrowser.gui.services.jobinterruption.JobInterruptionStore
 import com.spectralogic.dsbrowser.gui.services.tasks.Ds3JobTask
+import com.spectralogic.dsbrowser.util.exists
 import io.reactivex.functions.Action
 import io.reactivex.functions.Consumer
 import io.reactivex.rxjavafx.schedulers.JavaFxScheduler
+import javafx.application.Platform
 import javafx.beans.property.BooleanProperty
 import javafx.beans.property.SimpleBooleanProperty
+import javafx.concurrent.WorkerStateEvent
+import org.slf4j.Logger
+import java.io.IOException
 import java.util.*
 
 class JobTask(private val wrappedJob: JobFacade) : Ds3JobTask() {
     @Throws(Throwable::class)
     override fun executeJob() {
         ds3Client = wrappedJob.getDs3Client()
-        var throwable : Throwable? = null
+        var throwable: Throwable? = null
```

Comment:
I'm not sure that this logic for capturing a throwable is needed.  If this call is meant to block on the stream, make an explicit blocking call and the blocking will cause the throwable to get bubbled up.